### ISSUE TITLE
Anki 2.1.35 Cloze Deletion Bug?

### ISSUE THREAD

**Issue by SteamingTeapot** [Liresol/anki-custom-shortcuts#88]
Hello, 
It seems to me there can be some bug when it comes to editor cloze alt (i use Anki.2.1.35)
so the default is:
"editor cloze": "Ctrl+Shift+C",
"editor cloze alt": "Ctrl+Shift+Alt+C",
and i changed to
"editor cloze alt": "Ctrl+Shift+B",(and then once more to Ctrl+Shift+Q just to be more sure)
after doing it "cloze alt" behaved exactly like just cloze (giving c1, c2, etc), the default Ctrl+Shift+Alt+C does give c1, c1, etc. 
In essence, no shortcut seem to be able to  to overwrite the Ctrl+Shift+Alt+C 
also i disabled all addons but this one, so other addons dont interfere. And i also restarted anki every time i changed any shortcut. 
i also changed the reviewer _duplicates to 
  "reviewer _duplicates": {
        "reviewer choice 1": "j",
        "reviewer choice 2": "k",
        "reviewer choice 3": "l",
        "reviewer choice 4": ";"
    },
and that seem to work fine, (1,2,3,4 are still active thought but i think it  meant to be like that)

Could you help me please? 
thank you

**Comment by Liresol** [comment]
Hello,
Due to addon compatibility reasons (the old cloze alt behavior messed with other addons), `"editor cloze alt"` actually now just behaves the same way as `"editor cloze"`. 
However, you can get the functionality (c1, c1, c1...) with `"editor cloze no increment"`, so setting `"editor cloze no increment": "Ctrl+Shift+B"` should give you what you want.
Sorry for the inconvenience.

**Comment by SteamingTeapot** [comment]
hello,
thank you for reply. 
if i do 
"editor cloze": "Ctrl+Shift+C",
    "editor cloze alt": "<nop>",
    "editor cloze forced increment": "<nop>",
    "editor cloze no increment": "<Ctrl+Shift+B>",
then Ctrl+Shift+B does nothing and  Ctrl+Shift+Alt+C  (obviously) does noting 
i did also 
"editor cloze": "Ctrl+Shift+C",
    "editor cloze alt": "Ctrl+Shift+Alt+C",
    "editor cloze forced increment": "<nop>",
    "editor cloze no increment": "<Ctrl+Shift+B>",
Ctrl+Shift+B still does noting. Ctrl+Shift+Alt+C works as expected. 

i actually looked at this video by AnKing on the customise keyboard shortcuts and in the comment section (comment >3months), what u suggested fixes the issue but not for me. 

Godmode add on somehow works, so i will use that instead , i am left handed so i am not bothered about jkl;  :D

But i really appreciate that you answered, thank you. 
Best wishes,

**Comment by SteamingTeapot** [comment]
Hello,
the add-on works now (after the recent update). What happened is that I
tried what you wrote to me in the email before(i figured it out myself
shortly after i sent you the email) and the add-on still was not working.
However, I failed to notice that you did an update on 18.11 and you replied
on 19.11 and i tried  "editor cloze no increment": "Ctrl+Shift+B"  before
your update . I replied something like: it still does not work but never
mind, and i am sorry because it was fixed i just did not rechecked it.
Thank you so much and have a nice day,
Mag
